client/bio-webclient/src/main/kotlin/ac/uk/ebi/biostd/client/api/PermissionOperationsClient.kt

```diff
@@ -5,8 +5,8 @@ import org.springframework.web.client.RestTemplate
 
 class PermissionOperationsClient(private val restTemplate: RestTemplate) :
     PermissionOperations {
-    override fun givePermissionToUser(userEmail: String, accessTagName: String, accessType: String) {
-        val body = hashMapOf("userEmail" to userEmail, "accessType" to accessType, "accessTagName" to accessTagName)
+    override fun givePermissionToUser(user: String, accessTagName: String, accessType: String) {
```

Comment:
actually I think userEmail is more clear as here is ambiguous what can be , one may think is the id or the login